pyk4a/result.py

```diff
@@ -0,0 +1,8 @@
+from enum import IntEnum
+
+
+# k4a_wait_result_t
+class Result(IntEnum):
+    Success = 0
+    Failed = 1
+    Timeout = 2
```

Comment:
Should we move into errors.py since it's so small?

Reply:
Added two more error types from playback here.